src/pages/ProductList/ListMenu/ListMenu.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import './ListMenu.scss';
+
+export class ListMenu extends Component {
+  constructor() {
+    super();
+    this.state = {
+      menuList: [],
+    };
+  }
+  componentDidMount() {
+    fetch('http://localhost:3000/data/listMenu.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          menuList: data,
+        });
+      });
+  }
+  render() {
+    const { menuList } = this.state;
+
+    return (
+      <article className="listMenu">
+        <div className="listTitle">
+          <h2>샤워</h2>
+          <select name="hero">
+            <option value="신제품순">신제품순</option>
+            <option value="낮은가격순">낮은가격순</option>
+            <option value="높은가격순">높은가격순</option>
+          </select>
+        </div>
+        <ul className="listCategory">
+          <li>전체 (30)</li>
+          {menuList.map((el, i) => {
+            return (
+              <li key={i}>
+                {el.name} ({el.quantity})
+              </li>
+            );
+          })}
```

Comment:
map 메서드 잘 활용하셨습니다. el이 의미하는 매개변수를 좀 더 시멘틱하게 작성해주시면 좋을 것 같네요! 
```suggestion
          {menuList.map((menu, i) => {
            return (
              <li key={i}>
                {menu.name} ({menu.quantity})
              </li>
            );
          })}
```